src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -556,13 +556,37 @@ public function nodeName()
     }
 
     /**
+     * Returns the text of the first node of the list.
+     *
+     * @param bool $normalizeWhitespaces Pass true to normalize whitespaces, not passing true is deprecated since version 3.4.
+     *
+     * @return string The node value
+     *
+     * @throws \InvalidArgumentException When current node is empty
+     */
+    public function text($normalizeWhitespaces = false)
+    {
+        if (!$normalizeWhitespaces) {
+            @trigger_error(sprintf('Not normalizing whitespace characters when using the %s() method is deprecated since version 3.4. The %s() method will return the first node\'s value with whitespaces being normalized in Symfony 4.0. Use the value() method instead if you do not want to have whitespaces normalized.', __METHOD__, __METHOD__), E_USER_DEPRECATED);
```

Comment:
This deprecation is not required and should be removed.
Also, the added arg is a bc break right now, should be done using func_get_arg()